.kokoro/common.cfg

```diff
@@ -0,0 +1,23 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+build_file: "google-cloud-python/.kokoro/build.sh"
```

Comment:
remove `build_file` from here, and put it in `.kokoro/docs.cfg` and `s/build.sh/build-docs.sh`.